core/src/impl/Kokkos_Core.cpp

```diff
@@ -824,8 +824,14 @@ instead of hyphens). For example, to disable warning messages, you can either
 specify --kokkos-disable-warnings or set the KOKKOS_DISABLE_WARNINGS
 environment variable to yes.
 
-Join us on Slack, visit https://kokkosteam.slack.com
-Report bugs to https://github.com/kokkos/kokkos/issues
+--------------------------------------------------------------------------------
+For support, documentation, and more information about Kokkos,
+visit the official website: https://kokkos.org
+
+If you use Kokkos in your work, please cite the recommended papers
+found at https://kokkos.org/citing-kokkos in your science publications.
```

Comment:
```suggestion
Please cite the recommended publications listed at https://kokkos.org/citing-kokkos
when using Kokkos in your scientific work.
```

Reply:
I prefer the original message, it feels a little bit less injunctive imo
```suggestion
If you use Kokkos in your scientific work, please cite the recommended
publications found at https://kokkos.org/citing-kokkos.
```